src/software/thunderscope/robot_diagnostics/ssh_dialog.py

```diff
@@ -0,0 +1,98 @@
+from pyqtgraph.Qt.QtWidgets import *
+from software.embedded.constants.py_constants import RobotPlatform, NetworkConstants
+import software.thunderscope.common.common_widgets as common_widgets
+
+
+class SSHDialog(QDialog):
+    """Dialog prompting the user to enter the IP address/hostname and user password
+    of a robot to connect to via SSH.
+
+    The dialog also prompts the user to select the robot's compute module platform
+    so that, once connected via SSH, we can flash the robot with the appropriate
+    binaries or execute platform-specific tasks.
+    """
+
+    DEFAULT_SSH_PASSWORD = "thunderbots"
```

Comment:
we don't want to leak our secrets on the repo

Reply:
Lmao, we have leaked this through commit history

Reply:
whoops